Gemfile.lock

```diff
@@ -0,0 +1,200 @@
+GEM
```

Comment:
Hmm, I didn't pay it much attention really. All the Gemfile is specifying is the `github-pages` gem. Let me see if I can get it working with the bare bones suggestion